src/cosmicqc/cli.py

```diff
@@ -0,0 +1,111 @@
+"""
+Setup coSMicQC CLI through python-fire
+"""
+
+import inspect
+import sys
+import types
+from typing import Optional
+
+import fire
+from fire import helptext, inspectutils, value_types
+from fire.core import Display, FireError, _DictAsString, _OneLineResult
+from fire.trace import FireTrace
+
+from . import analyze
```

Comment:
What is this import for?

Reply:
Thanks for this question - this import helps include the `analyze` module (and as a result, the functions found within `analyze.py`) for use with the CLI through `python-fire`.